app/Console/Commands/TestCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class TestCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'command:name';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        eval($this->option('test'));
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Remove this use of the "eval" function. ([php:S1523](https://sonarcloud.io/organizations/dbmeneses-github/rules#rule_key=php%3AS1523))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=monica-dm-test&pullRequest=1&issues=AWSsghyll6vpliCetuqL&open=AWSsghyll6vpliCetuqL)

[ISSUE_KEY]: AWSsghyll6vpliCetuqL